lib/vtls/sectransp.c

```diff
@@ -1660,7 +1660,7 @@ static CURLcode sectransp_connect_step1(struct Curl_easy *data,
   struct ssl_connect_data *connssl = &conn->ssl[sockindex];
   struct ssl_backend_data *backend = connssl->backend;
   const char * const ssl_cafile = SSL_CONN_CONFIG(CAfile);
-  const struct curl_blob *ssl_cablob = NULL;
+  const struct curl_blob *ssl_cablob = SSL_CONN_CONFIG(ca_info_blob);
```

Comment:
I think you'll need the same check for ssl_cafile as you used in openssl `ssl_cafile = (ca_info_blob ? NULL : SSL_CONN_CONFIG(CAfile));`

Reply:
Yes, I just do it